Build/Scripts/Synchrophasor.buildproj

```diff
@@ -155,4 +155,7 @@
     <MakeDir Directories="$(HelpDeployFolder)" Condition="!Exists('$(HelpDeployFolder)')"/>
     <Copy SourceFiles="@(HelpFilesToDeploy)" DestinationFolder="$(HelpDeployFolder)\%(HelpFilesToDeploy.RecursiveDir)"/>
   </Target>
+    <Target Name="AfterPushToServer" Condition="'$(DoNotPush)' == 'false' And '$(SkipVersioning)' != 'true'">
+    <Exec Command="%22$(AfterPushScript)%22 $(ProjectName) $(BuildTag)" Condition="'$(PushToVersionBranch)' == 'true' And Exists('$(AfterPushScript)')" />
+  </Target>
```

Comment:
This one needs `openPDC` explicitly specified, since the `$(ProjectName)` variable resolves to `Synchrophasor` which doesn't match the name of the Git repository.

```suggestion
  <Target Name="AfterPushToServer" Condition="'$(DoNotPush)' == 'false' And '$(SkipVersioning)' != 'true'">
    <Exec Command="%22$(AfterPushScript)%22 openPDC $(BuildTag)" Condition="'$(PushToVersionBranch)' == 'true' And Exists('$(AfterPushScript)')" />
  </Target>
```